flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerUpdateImportServiceImpl.java

```diff
@@ -163,6 +169,12 @@ private FrontLineWorker flwFromRecord(Map<String, Object> record) {
         mapping.put(NMS_FLW_ID, new Optional(new GetString()));
         mapping.put(MCTS_FLW_ID, new Optional(new GetString()));
         mapping.put(MSISDN, new Optional(new GetLong()));
+        mapping.put(STATE, new GetInstanceByLong<State>() {
+            @Override
+            public State retrieve(Long value) {
+                return stateDataService.findByCode(value);
```

Comment:
Shouldn't `findByCode` be `findById`?
Don't we generally have codes be strings and ids numbers?